pkg/image/controller/trigger/image_trigger_controller_test.go

```diff
@@ -836,6 +883,28 @@ func scenario_1_buildConfig_imageSource_noImageIDChange() *buildv1.BuildConfig {
 	}
 }
 
+func scenario_1_buildConfig_imageSource_noImageIDChange_pre48Cluster() *buildv1.BuildConfig {
```

Comment:
How is this scenario different from the `scenario_1_buildConfig_imageSource_pre48Cluster` scenario?

Reply:
with this one the last triggered image ID is out of date / old 

`"image/result:1"` vs. `"image/result:2"`

I will also highlight this in the upcoming godoc

Reply:
will also try to better capture this in the method name

Reply:
sorry I have that backwards ... it will be sorted out in the update I push